modules/get-started/pages/rpk-first-steps.adoc

```diff
@@ -0,0 +1,241 @@
+= First Steps in rpk
+:page-categories: rpk
+// Do not put page aliases in the single-sourced content
+// tag::single-source[]
+
+This guide shows you how to use `rpk` to perform fundamental Redpanda tasks, including creating, producing to, describing, and deleting topics, as well as consuming records and managing consumer groups. Following these examples will help you quickly become more familiar with `rpk` commands and integrate them into your workflow.
```

Comment:
I think it looks great, the only thing I'll add is creating a profile as a first step, so they don't have to wonder how to run the command against a cluster.

Alternatively, we could say that you can try this tutorial using `rpk container start` which creates a profile automatically and links to the profile docs as additional documentation.

Back when we created the issue we didn't have profiles

Reply:
Yeah I was thinking about that. We have the create profile page before this one. But I think we could combine those into a single page. Specially for cloud